packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -261,3 +261,13 @@ The following is an example System Log rate limit event where too many OTP verif
   ]
 }
 ```
+
+## Operation rate limit subtypes
+
+The following table includes the available subtypes for operation rate limits.
+
+| Subtype                   | Description                                                                   | Rate limit |
+| ------------------------- | ----------------------------------------------------------------------------- | ---------- |
+| `email`                   | ?                                                                             |            |
+| `phone`                   | ?                                                                             |            |
+| `rsa_token`               | ?                                                                             |            |
```

Comment:
- `Email` : The user exceeded their limit for sending email messages
- `SMS` : The user exceeded their limit for sending sms messages
- `Voice call` : The user exceeded their limit for sending voice-call messages
- `rsa_token` : _Will update when I hunt it down_

I don't think we are being public with these rate limits.  Certainly not for sms or voice-call.  Will confirm.

Reply:
- `rsa_token` : Remove